bucket_A4/toxic/specification

```diff
@@ -0,0 +1,48 @@
+DEF[PORTVERSION]=	0.8.2
+# ----------------------------------------------------------------------------
+
+NAMEBASE=		toxic
+VERSION=		${PORTVERSION}
+KEYWORDS=		net net_im net_p2p
+VARIANTS=		standard
+SDESC[standard]=	Ncurses-based Tox client
+HOMEPAGE=		https://github.com/Jfreegman/toxic
+CONTACT=		Leonid_Bobrov[mazocomp@disroot.org]
+
+DOWNLOAD_GROUPS=	main
+SITES[main]=		GITHUB/JFreegman:toxic:v${PORTVERSION}
+DISTFILE[1]=		generated:main
+
+SPKGS[standard]=	single
+
+OPTIONS_AVAILABLE=	none
+OPTIONS_STANDARD=	none
+
+LICENSE=		GPLv3+:single
+LICENSE_FILE=		GPLv3+:{{WRKSRC}}/LICENSE
+LICENSE_TERMS=		single:{{WRKDIR}}/TERMS
+LICENSE_AWK=		TERMS:"^\#include"
+LICENSE_SOURCE=		TERMS:{{WRKSRC}}/src/toxic.c
+LICENSE_SCHEME=		solo
+
+FPC_EQUIVALENT=		net-im/toxic
+
+USES=			desktop-utils:single gmake ncurses pkgconfig python:py37
```

Comment:
You hardcoded python37.  This looks wrong.   It should probably be "python".  Don't specify a version unless there is a very good reason.